addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -295,8 +295,9 @@ def _cron_receive_fattura_pa(self):
         if self.env['account_edi_proxy_client.user']._get_demo_state() == 'demo':
             return
 
+        fattura_pa = self.env.ref('l10n_it_edi.edi_fatturaPA')
```

Comment:
It won't hurt, but it does not seem needed.  (just wanted to make sure I understand correctly)